CHANGELOG.next.asciidoc

```diff
@@ -98,6 +98,9 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Filebeat*
 
 - Fix using log_group_name_prefix in aws-cloudwatch input. {pull}29695[29695]
+- aws-s3: Improve gzip detection to avoid false negatives. {issue}29968[29968]
+- decode_cef: Fix panic when recovering from invalid CEF extensions that contain escape characters. {issue}30010[30010]
```

Comment:
@aspacca should this be here? Feels like they should not

Reply:
no